<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1295</width>
    <height>977</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_return">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>155</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_return">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_about">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>50</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_about">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>165</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_bottom">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>855</y>
     <width>700</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string>执行操作界面还剩             秒，即将退出操作返回首页</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>850</y>
     <width>80</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
